modules/ROOT/pages/introduction/intro-config-use-studio.adoc

```diff
@@ -0,0 +1,160 @@
+
+To use Scheduler as an input source:
+
+. In the Mule Palette view, search for and select *Scheduler*:
++
+.Select Scheduler to schedule when a flow is initiated
+image::introduction/intro-config-studio-select-scheduler.png[Selecting Scheduler as an input source]
++
+. Drag *Scheduler* onto the canvas.
++
+The following screenshot shows the default values for the scheduling strategy:
++
+.Default values for the scheduling strategy
+image::introduction/intro-config-studio-scheduler.png[Default values for the scheduling strategy]
++
+. In the *Scheduling Strategy* field of the *Scheduler configuration* screen, select whether you want to use a fixed frequency or cron expression to initiate the flow.
+* If you select *Fixed Frequency*, enter values for these fields:
++
+** *Frequency*
++
+The frequency at which the Scheduler initiates the flow, based on the time unit specified in the *Time unit* field
++
+** *Start delay*
++
+The amount of time to wait before initiating the flow for the first time after the app starts, based on the time unit specified in the *Time unit* field
++
+** *Time unit*
++
+The time unit for the values of *Frequency* and *Start Delay*
++
+* If you select *Cron*, enter the cron expression in the *Expression* field and the time zone in the *Time Zone* field. For
+ the format and valid values for the *Time Zone* field, see the Javadoc for the Java TimeZone class.
++
+. To configure metadata for the payload, variables, and attributes, select the *Metadata* tab and add the appropriate metadata.
+
+For more information about Scheduler, see xref:mule-runtime::scheduler-concept.adoc[Scheduler Endpoint (Trigger)].
+
+=== Use a Connector-Specific Input Source
+
+If a connector has associated input sources, you can use a connector-specific input source to initiate the flow. Each connector input source has a unique configuration.
+
+For example, you can use Amazon SQS Receive Messages as an input source for Amazon SQS Connector. This input source initiates a flow each time the specified Amazon SQS queue receives a new message.
+
+To configure a connector-specific input source:
+
+. In the Mule Palette view, search for the connector and select the input source.
++
+For example, search for *Amazon SQS* and select the *Receive Messages* input source:
++
+.Searching for the Amazon SQS Receive Messages input source
+image::introduction/intro-config-studio-select-specific.png[Amazon SQS Receive Messages input source]
++
+. Drag the input source onto the canvas.
+. In the configuration screen, click the plus sign (*+*) next to the *Connector configuration* field to add a global element for the input source.
+. Enter values for the required fields.
+. On the *Advanced* tab next to the *Proxy* tab, specify a reconnection strategy.
```

Comment:
If the audience can readily find the Proxy tab, why could it not find the Advanced tab? Not sure how naming something next to another helps to locate it....